StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/ViewControllers/UpdatePaymentMethodViewController+Configuration.swift

```diff
@@ -50,6 +51,9 @@ extension UpdatePaymentMethodViewController {
         var footnote: String? {
             switch paymentMethod.type {
             case .card:
+                if canUpdate {
```

Comment:
Nit: Guard feels better here

Reply:
I think i disagree.  A guard would result in:
```
guard !canUpdate else {
   return nil
}
return canUpdateCardBrand ? .Localized.only_card_brand_can_be_changed : .Localized.card_details_cannot_be_changed

```

The way this reads feels awkward because you are ensuring that you can not update, and then immediately read "can update card brand".